lua/telescope/_extensions/coc.lua

```diff
@@ -382,7 +382,7 @@ local diagnostics = function(opts)
     end
   end
 
-  opts.hide_filename = utils.get_default(opts.hide_filename, true)
+  opts.path_display = utils.get_default(opts.path_display, "shorten")
```

Comment:
Maybe `hidden` is better, because these diagnostics are limited in current buffer, we can hide the file path.